src/main/scala/sbtcompatibility/SbtCompatibilityMima.scala

```diff
@@ -1,19 +1,93 @@
 package sbtcompatibility
 
 import com.typesafe.tools.mima.plugin.MimaPlugin
-import sbt.{AutoPlugin, Def, Keys}
-import sbtcompatibility.internal.Version
+import coursier.version.Previous
+import sbt.{AutoPlugin, Def, Keys, settingKey}
+
+import scala.collection.JavaConverters._
+import coursier.version.VersionCompatibility
+import coursier.version.Version
+import sbt.librarymanagement.CrossVersion
+import sbt.librarymanagement.ModuleID
 
 object SbtCompatibilityMima extends AutoPlugin {
 
   override def trigger = allRequirements
   override def requires = MimaPlugin
 
+  object autoImport {
+    val previousVersions = settingKey[Seq[String]]("")
+    val firstVersion = settingKey[Option[String]]("")
```

Comment:
Could you add a sensible description of these keys instead of passing an empty string?

Reply:
Fixed in https://github.com/scalacenter/sbt-compatibility/pull/24.